consensus.asciidoc

```diff
@@ -4,8 +4,12 @@
 
 When it comes to decentralized record keeping, which is how blockchain technology is often described, it becomes infeasible to rely on trust alone to ensure that factual information is added to the record books. This problem arises in decentralized networks as a result of the lack of a central entity that would typically decide what should and should not be considered as fact. It's difficult to avoid this issue when working with blockchain technology because it is this lack of a central entity that is one of the main contributors to blockchain's popularity. In order to circumvent this issue a network must have a system in place by which a consensus can be reached by the nodes in the network.
 
+//Philosofical: In effect, you are trusting the system. I never appreciated the idea of a "trustless" system, instead one might look at it as shifting the trust in someone to trust in the system.
+
 A consensus protocol is a system that allows network nodes to reach an agreement on what should be considered as truth. When referring to a blockchain network this protocol is used to determine which copy of the blockchain is the true copy. This is required in order to prevent malicious parties from frauding the network and to guarantee the integrity of the blockchain is maintained. There are different types of consensus protocols in use in blockchain technologies, we will focus on proof of work (*POW*) and proof of stake (*POS*).
 
+//I am probably not suited to contribute and write about Ethereum consensus, but as a reader I would like to read more about how consensus work to provide immutability, before entering into POW/POS and specifics. I understand this is a work in progress.
```

Comment:
Just a little note about such a trancendental point. For me, through the **"Vlad Zamfir's"** works, it was fascinating to discover that there is a branch of engineering specialized in Consensus, something that surely those deeply interested in the subject should study and cultivate.

Reply:
Will definitely look into it, thanks.